mmeds/tests/metadata/test_error_metadata.py

```diff
@@ -0,0 +1,96 @@
+from unittest import TestCase
+import mmeds.validate as valid
+import mmeds.config as fig
+import sys
+import traceback as tb
+from pathlib import Path
+
+# Set as True to see formatted error output
+VERBOSE = False
+
+class ErrorTests(TestCase):
+    def test_a_generated_error_files(self):
+        """
+        Test all the error files generated by scripts/generate_test_tsv.py
+        """
+        
+        top_dir = Path('/home/adamcantor22//mmeds-meta/test_files/validation_files/')
+        sub_directories = ['blank_column_tests', 'na_column_tests', 'other_column_tests', 'number_column_tests', 'date_column_tests']
+
+        # Get all sub directories to analyze
+        total_directories = []
+        for directory in sub_directories:
+            total_directories.append(top_dir / directory / 'subject')
+            total_directories.append(top_dir / directory / 'specimen')
```

Comment:
You can use `glob` here. It finds files using regular expresssions
```suggestion
       total_directories = list(top_dir.glob('*/subject/')) + list(top_dir.glob('*/specimen/'))
```